.github/workflows/komodo_linux_ci.yml

```diff
@@ -247,32 +224,41 @@ jobs:
           cd qa/pytest_komodo
           ./ci_setup.sh cc_modules/test_heir.py
 
-  windows-build:
-
-    name: Win Build
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v1
-    - name: Build Win on Unix
-      env:
-        DEBIAN_FRONTEND: noninteractive
-      if: runner.os == 'Linux'
-      run: |
-        sudo apt-get remove php5.6-fpm php7.0-fpm php7.1-fpm php7.2-fpm php7.3-fpm php7.3-common php7.4-fpm msodbcsql17
-        sudo apt-get update
-        sudo apt-get upgrade -y
-        sudo apt-get install build-essential pkg-config libc6-dev m4 g++-multilib autoconf libtool libncurses-dev unzip git python zlib1g-dev wget bsdmainutils automake libboost-all-dev libssl-dev libprotobuf-dev protobuf-compiler libqrencode-dev libdb++-dev ntp ntpdate nano software-properties-common curl libevent-dev libcurl4-gnutls-dev cmake clang libsodium-dev -y
-        sudo apt-get install build-essential pkg-config libc6-dev m4 g++-multilib autoconf libtool ncurses-dev unzip git python python-zmq zlib1g-dev wget libcurl4-gnutls-dev bsdmainutils automake curl cmake mingw-w64
-        curl https://sh.rustup.rs -sSf | sh -s -- -y
-        source $HOME/.cargo/env
-        rustup target add x86_64-pc-windows-gnu
-        sudo update-alternatives --set x86_64-w64-mingw32-gcc /usr/bin/x86_64-w64-mingw32-gcc-posix
-        sudo update-alternatives --set x86_64-w64-mingw32-g++ /usr/bin/x86_64-w64-mingw32-g++-posix
-        CONFIGURE_FLAGS='CPPFLAGS=-DTESTMODE' ./zcutil/build-win.sh -j$(nproc)
-        zip --junk-paths komodod_win src/komodod.exe
-    - name: Upload komodod.exe as artifact
-      uses: actions/upload-artifact@v1
-      with:
-        name: komodod_win
-        path: ./komodod_win.zip
+#  linux-test-dexp2p:
```

Comment:
@SirSevenG  is dexp2p broken?

Reply:
Was dexp2p merged to dev already?

Reply:
@tonymorony Just re-checked - dexp2p is not in dev branch yet.